vtex/address-form
# CHK-1969 - CHK-1968 - (V4) Address review flow is removing essential address information and showing calculate shipping button for complete address

#### What is the purpose of this pull request?

Fix "Calculate Shipping" button working incorrectly by fixing [this](https://vtex-dev.atlassian.net/browse/CHK-1969) and [this](https://vtex-dev.atlassian.net/browse/CHK-1968) KIs.

#### What problem is this solving?

Fix the issue that the "Calculate Shipping" button was showing up even for full addresses, which caused 2 issues:

- It made it impossible for the user to proceed with the payment
- Caused some information to be lost when the user clicked on the "Calculate Shipping" button

This problem was happening in stores in France, but it had already been fixed for stores in Brazil and Chile previously.

#### How should this be manually tested?

- Access https://vtexgame1geo.myvtex.com/checkout/cart/add/?sku=312&qty=1&seller=1&sc=2
- In the Shipping step, select France and enter the address `42000 Saint-Étienne`
- The address will change, the "Calculate shipping" button should not be displayed and it should be possible to proceed to the payment step as normal


#### Types of changes
